test/legacy/fairroot.sh.in

```diff
@@ -8,7 +8,8 @@ version="@TEST_VERSION@"
 sourcedir=FairRoot_${version}
 git clone -b ${version} https://github.com/FairRootGroup/FairRoot $sourcedir
 pushd $sourcedir
-export FAIRROOTPATH="$(realpath ./install)"
+mkdir -p build/install
+export FAIRROOTPATH="$(realpath ./build/install)"
```

Comment:
NOTE about moving the install tree inside the build tree:

Installing into the build tree is a common idiom.

If we adopt something like https://github.com/vmc-project/geant4_vmc/commit/12bc85eec9d654e24be8571b7ca31c8f7e628707 for FairRoot (which we should at some point for security and portability reasons), then the `-excludePath "${CMAKE_CURRENT_BINARY_DIR}"` might remove the install path as well from the search path inside the dictionary. Of course, setting `ROOT_INCLUDE_PATH` will mitigate this.